docs/developer/gift-cards.mdx

```diff
@@ -355,94 +179,33 @@ mutation {
-            "type": "DEACTIVATED"
-          },
-          {
-            "type": "ACTIVATED"
-          }
-        ]
-      }
-    }
-  }
-}
-```
+For bulk operations, use [`giftCardBulkActivate`](api-reference/gift-cards/mutations/gift-card-bulk-activate.mdx) and [`giftCardBulkDeactivate`](api-reference/gift-cards/mutations/gift-card-bulk-deactivate.mdx) mutations.
 
-For bulk activation, use [giftCardBulkActivate](api-reference/gift-cards/mutations/gift-card-bulk-activate.mdx)
-and for bulk deactivation use [giftCardBulkDeactivate](api-reference/gift-cards/mutations/gift-card-bulk-deactivate.mdx) mutation.
+## Setting Up Gift Cards as Products
 
-## Creating gift card products
+To allow customers to purchase gift cards, you need to:
 
-To allow customers to purchase gift cards, you have to [create a product type](api-reference/products/mutations/product-type-create.mdx) with `GIFT_CARD` kind and use it to [create a product](api-reference/products/mutations/product-create.mdx).
+1. Create a product type with `GIFT_CARD` kind
+2. Create a product using this product type
+3. Configure stock settings
 
 :::note
-If you want to have an unlimited number of gift cards in your shop you should create a stock in a chosen channel with at least one quantity and unset track inventory flag.
+If you want to have an unlimited number of gift cards in your shop, you should create a stock in a chosen channel with at least one quantity and unset the track inventory flag.
 :::
 
-# Buying gift cards
+### Gift Card Fulfillment Settings
+
+The `automaticallyFulfillNonShippableGiftCard` order setting controls when bought gift cards will be created:
+- If set to `True`: Gift card is created during checkout completion
+- If set to `False`: Gift card is created during order fulfillment
 
-The `automaticallyFulfillNonShippableGiftCard` order setting defines when the bought gift cards will be created. If this flag is set to `True`, a gift card will be created during checkout completion.
-The fulfillment will be created automatically, and a gift card will be generated and sent to the customer.
-If this option is unset, the gift card will be created during order fulfillment and, after that, also sent to the customer.
-The value of this option can be changed with use of [orderSettingsUpdate](api-reference/orders/mutations/order-settings-update.mdx) mutation.
+### Gift Card Expiry Settings
 
-The created gift card expiry date is set based on gift card settings.
-Gift card settings can be set never to expire or set to an expiry period, based on which the card expiry date is calculated.
+Gift card settings can be configured to:
+- Never expire
+- Expire after a specific period
 
-The gift card settings are set as never expiry by default and can be changed with use of [giftCardSettingsUpdate](api-reference/gift-cards/mutations/gift-card-settings-update.mdx).
-In the following example, the gif card settings are set to 12 months.
+Example of setting expiry period to 12 months:
```

Comment:
Mutation has expiry type `YEAR` - which means 12 years expiry period. Either mutation or description should be changed